blazor/query-builder/getting-started.md

```diff
@@ -1,22 +1,22 @@
 ---
 layout: post
-title: Getting Started with Blazor QueryBuilder Component | Syncfusion
-description: Checkout and learn about getting started with Blazor QueryBuilder component in Blazor Server App and Blazor WebAssembly App.
+title: Getting Started with Blazor Query Builder Component | Syncfusion
+description: Checkout and learn about getting started with Blazor Query Builder component in Blazor Server App and Blazor WebAssembly App.
 platform: Blazor
-control: QueryBuilder
+control: Query Builder
 documentation: ug
 ---
 
-# Getting Started with Blazor QueryBuilder Component
+# Getting Started with Blazor Query Builder Component
 
-This section briefly explains about how to include [Blazor QueryBuilder](https://www.syncfusion.com/blazor-components/blazor-query-builder) component in your Blazor Server App and Blazor WebAssembly App using Visual Studio.
+This section briefly explains about how to include [Blazor Query Builder](https://www.syncfusion.com/blazor-components/blazor-query-builder) component in your Blazor Server App and Blazor WebAssembly App using Visual Studio.
 
-To get started quickly with Blazor QueryBuilder, check on the following video or [GitHub](https://github.com/SyncfusionExamples/Blazor-Getting-Started-Examples/tree/main/QueryBuilder) sample:
+To get started quickly with Blazor Query Builder, check on the following video or [GitHub](https://github.com/SyncfusionExamples/Blazor-Getting-Started-Examples/tree/main/Query Builder) sample:
```

Comment:
This is the link, why you changed the control name?

Reply:
reverted